### DISCUSSION TITLE
Create Account JWT from Java Client

### DISCUSSION THREAD

**Discussion by sharma-abhishek2023** [nats-io/nats.java#576] [Q&A]
I am referencing this function to create a JWT for an account [issueUserJWT](https://github.com/nats-io/nats.java/blob/main/src/main/java/io/nats/client/support/JwtUtils.java#L176), but when I do this and push newly created jwt I get an error as **jwt update resulted in error - n/a - claim failed V2 signature verification** , Is this expected behavior or, Is there another way to create JWT for an account from client-side. Thanks

**Comment by scottf** [comment]
@sharma-abhishek2023 converted to discussion
